akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -439,11 +439,46 @@ object Attributes {
       strategy: CancellationStrategy.Strategy): CancellationStrategy.Strategy =
     CancellationStrategy.AfterDelay(delay, strategy)
 
-  case class NestedMaterializationCancellationPolicy(delay: Boolean) extends MandatoryAttribute
+  /**
+   * Nested materialization cancellation strategy provides a way to configure the cancellation behavior of stages that materialize a nested flow at some stage.
```

Comment:
```suggestion
   * Nested materialization cancellation strategy provides a way to configure the cancellation behavior of stages that materialize a nested flow.
```

Reply:
@seglo, the whole point here is that these stages may delay the nested flow's materialization - otherwise we wouldn't have to introduce this attribute.
don't u think the docs should some how mention this?

Reply:
Yes. I thought the trailing part of the sentence was redundant. You don't think it reads the same?

Reply:
@seglo , the attributes are separate from the actual operators they affect, hence a user reading this documentation may be confused.
I think this doc should provide the user with as much context as possible so she'd be able to understand the attribute's role.